app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> None:
+        return (
+            f"{{Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
+        )
+
+    @classmethod
+    def remove_dead(cls) -> None:
+        cls.alive = [animal for animal in cls.alive if animal.health > 0]
+
+    @classmethod
+    def __str__(cls) -> None:
+        return str([repr(animal) for animal in cls.alive])
```

Comment:
This method should not be a class method, as it logically should be used to provide a string representation of an instance of the Animal class. In your code, __str__ is written as a class method, but it does not provide a correct string representation for an instance of the class.
```suggestion
    @classmethod
    def __str__(self) -> str:
        return repr(self)
```